homeassistant/components/tuya/number.py

```diff
@@ -260,6 +260,29 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # EV charger
```

Comment:
Please point to the API documentation for this device category in the comment here.